README.md

```diff
@@ -7,3 +7,89 @@ Validate a JS object against a GraphQL schema
 
 Currently, this only checks that required properties are present. It does not check that the type of each property is correct, nor does it run custom validations for user-created types.
 
+
+## Usage
+
+```js
+const rootType = 'Person';
+const schema = `
+  type ${rootType} {
+    name: String!
+    knowsJS: Boolean!
+    age: Int
+    height: Float
+  }
+`;
+const validPerson = {
+  name: 'Alice',
+  knowsJS: true,
+  age: 42
+};
+const invalidPerson = {
+  knowsJS: "Yes"
+  age: 10.5,
+};
+validate(schema, rootType, validPerson);
+// []
+validate(schema, rootType)(invalidPerson);
+// [
+//   "'name' is required",
+//   "'knowsJS' must be of type boolean, received string",
+//   "'age' must be of type integer, received float"
+// ]
+```
+
+By default `validate` understands the native [GraphQL scalars](https://graphql.org/learn/schema/#scalar-types). It expects `ID` to be [cuids](https://github.com/ericelliott/cuid). See `configureGqlValidate` for supporting custom types or to change the validation of existing types.
```

Comment:
Why does `ID` have to be a cuid? Why not a hash? Why not any string?

Reply:
We need some default validation and I chose cuid because we use that all the time. We can have it to by default expect any string or hash or whatever you prefer. It just felt like cuids were the most sane default for our projects :)

Reply:
Just a string will suffice. We also very frequently use hashes for ids...

Reply:
Got it. I pushed an update clarifying that ID is just a string by default and that the user will have to handle special ID validation themselves.